app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,5 @@
+/*
+ *= require_tree .
+ *= require social-share-button
+ *= require_self
+ */
```

Comment:
Incidentally as I said I don't think anything will actually load `application.css` so putting it there will achieve nothing.